### PR TITLE
Fix-7 Added Snippets for Components with interfaces

### LINKED ISSUE #7: Add snippets for component using interfaces for states and props
@chetanraj , Can I add a new snippet for components using interfaces for states and props

Example snippet:

```
interface exampleState {
state1 : string;
state2: boolean;
state3 : any;
}

interface exampleProps {
props1: string;
props2: boolean;
prop3: any;
}

class ExampleComponent Extends React.Component<exampleProps,exampleState> {
constructor(props) {
    super(props);
    this.state = {
      state1: "",
      state2: false,
      state3: () => {}
    };
}
```

### PR DESCRIPTION
**Pull request by skprabhanjan**
@chetanraj , Here is the PR to fix #7 

PS : Feel free to close the issue and PR if you feel this is not required.

### CODE REVIEW

File: src/utils/Constants.js [hunk 1]
```diff
@@ -18,6 +18,32 @@ export default class Snpt extends Component {
     )
   }
 }`;
+
+const __ReactComponentWithInterfaces = `
+import React as * from 'react';
+
+interface exampleState {
+  state1 : string;
```
Comment (chetanraj) [inline src/utils/Constants.js]: can you please check the alignment

File: src/utils/Constants.js [hunk 2]
```diff
@@ -18,6 +18,32 @@ export default class Snpt extends Component {
...
+  state2: boolean;
+  state3 : any;
+  }
+  
+  interface exampleProps {
+  props1: string;
```
Comment (chetanraj) [inline src/utils/Constants.js]: can you please check the alignment

File: src/utils/Constants.js [hunk 3]
```diff
@@ -18,6 +18,32 @@ export default class Snpt extends Component {
...
+  props2: boolean;
+  prop3: any;
+  }
+  
+  class ExampleComponent extends React.Component<exampleProps,exampleState> {
+  constructor(props) {
```
Comment (chetanraj) [inline src/utils/Constants.js]: can you please check the alignment

**Comment by skprabhanjan** [comment]
@chetanraj , All Alignment issues are fixed now :)

**Comment by chetanraj** [comment]
Will take this in future.

**Pull request closed without merging**